flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
I never tested and build my application on other arch types yet.

Reply:
if you never do then it will never happen.